commit c94fcb2b120b468bb7758d8d85fbc65b925ca6f5 (refs/changes/22/3922/2)
Author: Mike Gerdts <mike.gerdts@joyent.com>
Date:   2018-05-09T13:06:21+00:00 (1 year, 5 months ago)
    
    OS-6802 zoneadmd child process pollutes environment (fix DEBUG)
    Reviewed by: John Levon <john.levon@joyent.com>
    Reviewed by: Dan McDonald <danmcd@joyent.com>
    Approved by: Dan McDonald <danmcd@joyent.com>

diff --git a/usr/src/cmd/zoneadmd/zoneadmd.c b/usr/src/cmd/zoneadmd/zoneadmd.c
index 4599453d86..21c6460eeb 100644
--- a/usr/src/cmd/zoneadmd/zoneadmd.c
+++ b/usr/src/cmd/zoneadmd/zoneadmd.c
@@ -1039,7 +1039,6 @@ do_subproc(zlog_t *zlogp, char *cmdbuf, char **retstr, boolean_t debug)
 
 	if ((child = fork()) == 0) {
 		int in;
-		struct sigaction sa = { 0 };
 
 		/*
 		 * SIGINT is currently ignored.  It probably shouldn't be so
